deployment/skiperator.kartverket.no_applications.yaml

```diff
@@ -234,6 +234,18 @@ spec:
                   - mountPath
                   type: object
                 type: array
+              gcp:
+                properties:
+                  auth:
+                    properties:
+                      serviceAccount:
+                        type: string
+                    required:
+                    - serviceAccount
+                    type: object
+                required:
+                - auth
+                type: object
```

Comment:
We need to write documentation in the README for the changed API

Reply:
Added field to readme and created confluence site.